packages/workspace/test/workspace/reference_indexes.test.ts

```diff
@@ -515,8 +516,10 @@ describe('updateReferenceIndexes', () => {
     })
 
     describe('When indexes are out of date', () => {
+      let mockedGetCustomReferences: jest.MockedFunction<ReferenceIndexesGetCustomReferencesFunc>
       beforeEach(async () => {
         await elementsSource.set(instance)
+        mockedGetCustomReferences = jest.fn().mockResolvedValue([])
```

Comment:
should use typed mock functions
```suggestion
        mockedGetCustomReferences = jest.fn().mockResolvedValue([])
```
```suggestion
        mockedGetCustomReferences = mockFunction<typeof mockedGetCustomReferences>().mockResolvedValue([])
```

Reply:
Not sure this is required, considering the assignment will fail if the `mockResolvedValue` type is not matching